### ISSUE TITLE
v1.0.0 is broken when installed using Dep

### ISSUE THREAD

**Issue by 0xbkt** [anacrolix/torrent#312]
FYI, with Dep, it uses `anacrolix/log@v0.2.0` and it does not have some functions needed by `anacrolix/torrent@v1.0.0`. It's better to bump a version because Dep fails with current version unless `master` branch is used.

Error:
```
# github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/torrent
vendor/github.com/anacrolix/torrent/client.go:203:5: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:322:5: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:324:4: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:436:4: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:671:4: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:848:3: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/client.go:1163:4: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/connection.go:1196:4: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/connection.go:1338:3: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/torrent.go:271:3: undefined: "github.com/0xbkt/torrentTest/vendor/github.com/anacrolix/log".Printf
vendor/github.com/anacrolix/torrent/torrent.go:271:3: too many errors
```

Sourcegraph (`repo:^github\.com/anacrolix/torrent$@v1.0.0 anacrolix/log`)
https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/anacrolix/torrent%24%40v1.0.0+anacrolix/log

I don't know the reason why Dep doesn't honor the pseudo version in `go.sum` file in the link above.

**Comment by WinPooh32** [comment]
I have the same issue using **go.mod**:
```
...
go: downloading github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db
go: downloading github.com/philhofer/fwd v1.0.0
# github.com/anacrolix/torrent
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:203:5: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:322:5: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:324:4: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:436:4: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:671:4: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:848:3: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/client.go:1163:4: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/connection.go:1196:4: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/connection.go:1338:3: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/torrent.go:271:3: undefined: "github.com/anacrolix/log".Printf
../../../../pkg/mod/github.com/anacrolix/torrent@v1.0.0/torrent.go:271:3: too many errors
```

Go version: `go version go1.11.5 linux/amd64`

It looks there is a bug in go build system. It finds collision with golang's std package `log`

**Comment by WinPooh32** [comment]
@0xbkt, I have solved this issue by using [govendor](https://github.com/kardianos/govendor) tool.

**Comment by anacrolix** [comment]
This should be fixed in the tag v1.0.1.
